scripts/bundleSizeConfig.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
   MIN_SIZE: 647,
-  MAX_SIZE: 689,
+  MAX_SIZE: 693,
```

Comment:
Jenkins is complaining about the `Bundle size for Tamil is too large at 694 kB`.
```suggestion
  MAX_SIZE: 695,
```